README.md

```diff
@@ -89,6 +89,7 @@ AI Provider can be configured by using CLI arguments or Environment variables.
 | [OpenAI](https://platform.openai.com/docs/guides/text-generation/chat-completions-api) `openai`                    | Yes             | `gpt-4o`, `gpt-4o-mini`, `gpt-4`, `gpt-3.5-turbo` (default: `gpt-4o-mini`)                  |
 | [Claude](https://claude.ai/new) `claude`                                                                     | Yes             | [see list](https://docs.anthropic.com/en/docs/about-claude/models#model-names) (default: `claude-3-5-sonnet-20241022`) |                                                                                |
 | [Ollama](https://github.com/ollama/ollama) `ollama`                                                                     | No (local)             | [see list](https://github.com/ollama/ollama/blob/main/docs/api.md#model-names) (required) |                                                                                |
+| [OpenRouter](https://openrouter.ai/) `openrouter`                                                                     | Yes             | `anthropic/claude-3.5-sonnet` (default), `anthropic/claude-3-5-haiku`, `openai/gpt-4o-mini` |                                                                                |
```

Comment:
Are only these 2 models supported?

Reply:
every openai and anthropic model is supported

https://openrouter.ai/models?q=anthropic
https://openrouter.ai/models?q=openai

Reply:
In that case I think it will be better to provide a link to see the list of all available models